### ISSUE TITLE
UI: (4.D) Inventory Page: Keyboard User Graphic Missing (CRASM-351)

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#429]
# 🐛 Summary #

There is no visible indication of focus when focus is on any of the links displayed in the main content of the Search Results page. Provide a visible border or outline around the links so that keyboard users know which link they are viewing.
Provide a visible indicator of focus such as a border or outline around links, menu items, and other actionable elements when they receive focus.

## Expected behavior ##
- A keyboard user should see visible indicator of focus around the Search Results cards.

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/d36adfdf-bdec-45b8-b2b2-fba0f3af2c23)
